python/llm/example/gpu/qlora_finetuning/README.md

```diff
@@ -0,0 +1,50 @@
+# Q-Lora (experimental support)
+
+This example demonstrate how to finetune a llama2-7b model use Big-LLM 4bit optimizations using [Intel GPUs](../README.md).
+
+## 0. Requirements
+To run these examples with BigDL-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+## Example: Finetune llama2-7b using qlora
+
+This example is ported from [bnb-4bit-training](https://colab.research.google.com/drive/1VoYNfYDKcKRQRor98Zbf2-9VQTtGJ24k?usp=sharing)
+
+### 1. Install
+
+```bash
+conda create -n llm python=3.9
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.0.110+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+pip install git+https://github.com/huggingface/transformers.git@95fe0f5
+pip install peft
```

Comment:
need to specify the specific version of `peft` (and shall we add it to the dependency?)

Reply:
I added the `perf` version. I think we can add `peft` to dependency when we can get a stable release `transformers` and then we can add/update them together.